README.rst

```diff
@@ -547,7 +547,7 @@ Companies using web3j
 Please submit a pull request if you wish to include your company on the list:
 
 - `Amberdata <https://www.amberdata.io/>`_
-- `blk.io <https://blk.io>`_
+- `web3labs.io <https://www.web3labs.com/`_
```

Comment:
this link is broken, should be `web3labs.com`

Reply:
Fixed it.